input/system/system.go

```diff
@@ -46,8 +46,18 @@ func GetSystemState(config config.ServerConfig, logger *util.Logger) (system sta
 		// runs on the database server itself and can gather local statistics
 		system = selfhosted.GetSystemState(config, logger)
 		system.Info.Type = state.CrunchyBridgeSystem
+	} else if config.SystemType == "aiven" {
+		system.Info.Type = state.AivenSystem
 	} else if dbHost == "" || dbHost == "localhost" || dbHost == "127.0.0.1" || config.AlwaysCollectSystemData {
 		system = selfhosted.GetSystemState(config, logger)
+	} else {
+		if globalCollectionOpts.TestRun {
+			// Detected as self hosted, but not collecting system state as we
+			// didn't detect the collector is running on the same instance as
+			// the database server.
+			// Leave logs for if this is a test run.
+			logger.PrintInfo("Remote host (%s) was specified for the database address, skip collecting system state. Consider enabling always_collect_system_data if it's a local database with a non-local IP address", dbHost)
```

Comment:
We're not super-consistent about error message format, but we generally put the thing that's wrong first, and then an explanation. Also, I think "local database with a non-local IP address" is somewhat confusing, especially since we are not very thorough on checking local connections above (e.g., we ignore IPv6 entirely, and all but one IPv4 loopback address https://superuser.com/questions/393700/what-is-the-127-0-0-2-ip-address-for ).

How about

```suggestion
			logger.PrintInfo("Skipping collection of system state: remote host (%s) was specified for the database address. Consider enabling always_collect_system_data if the database is running on the same system as the collector", dbHost)
```

Reply:
Nice, your suggestion is much better, will apply.